docs/docs/clients/overview.md

```diff
@@ -147,6 +147,12 @@ specific user and subsequently adding a segment override for that segment.
 
 :::
 
+## Local Evaluation Runtime Considerations
+
+When running in Local Evaluation mode, our SDKs expect to be run as long-lived processes. Serverless platforms like AWS
+Lambda either break this contract or make it much more complicated. As a result, we do not recommend running our SDKs in
+Local Evaluation mode on top of platforms like Lambda where you do not have complete control over process lifetimes.
```

Comment:
Should we recommend an alternative, like deploying an Edge Proxy in their Lamda's region?

Reply:
Good shout! Updated/